helm/libero-reviewer/templates/configmap-client.yaml

```diff
@@ -18,7 +18,7 @@ data:
         listen 9000;
 
         location /graphql {
-          proxy_pass http://{{ include "libero-reviewer.fullname" . }}-server:3000;
+          proxy_pass http://{{ include "libero-reviewer.fullname" . }}-submission:3000;
```

Comment:
This corresponds to a DNS name `libero-reviewer--test-submission[.default.svc.cluster.internal]` which is the `Service` name. Since all releases of all charts of all products are in the same namespace of the Kubernetes cluster, these values always need the `fullname` to avoid clashes: just `submission` would make `libero-reviewer--end2end` and `libero-reviewer--staging` clash.